source/server/admin/stats_render.h

```diff
@@ -95,5 +97,54 @@ class StatsJsonRender : public StatsRender {
   std::string value_buffer_; // Used for Json::sanitize for text-readout values.
 };
 
+// Implements the Render interface for textual representation of Prometheus stats
+// (see: https://prometheus.io/docs/concepts/data_model)
+class PrometheusStatsRender : public StatsRender {
```

Comment:
Perhaps it would make more sense to have `GroupedStatsRender` and `UngroupedStatsRender`, but IMO we could leave that to a future refactoring.

Reply:
I think what you have now is good. I feel like "Prometheus" vs (say) "Html" or "Text" is one dimension, and "Grouped" vs "Ungrouped" is orthogonal.